modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -116,7 +116,7 @@ namespace {{packageName}}.Client
 
         // Creates and sets up a RestRequest prior to a call.
         private RestRequest PrepareRequest(
-            String path, {{^netStandard}}RestSharp.{{/netStandard}}Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
>  we were holding off on #2790 until version 2.3.0 because of the breaking change in the method signature

Yup, I think that's why @markebbers filed this change against 2.3.0 branch.

> Do you think it would make sense to provide an overloaded signature with the older Dictionary type to avoid breaking consumers that rely on this method?

Yup, but I would suggest we consider it if there are demands from our users.